resources/server_install.rb

```diff
@@ -340,6 +340,12 @@
     notifies :run, 'ruby_block[wait for rundeckd startup]', :immediately
   end
 
+  execute 'enable rundeckd' do
+    command 'systemctl enable rundeckd'
+    action :run
+    only_if { node['init_package'] == 'systemd' }
+  end
```

Comment:
This should use the `service` resource instead of `execute`.

Reply:
@ramereth when we tried service is was reporting that the service was enabled but it was still disabled in systemd. This was the work around to fix it.

Reply:
@lamont-granquist / @tas50  recommendations for this?

Reply:
Building on what @codayblue said, this only occurs on Debian distros.